test/helper/formatCertificate.js

```diff
@@ -0,0 +1,40 @@
+module.exports = function formatCertificate (txJson) {
```

Comment:
why we need to format expected test data in a different way? does this mean we have different json format between zencashjs and bitcore-lib-zen? maybe we should investigate if it is easy to unify them

Reply:
@paolocappelletti 
The `inputs` and `outputs` were always in a different format between zencashjs and bitcore-lib-zen, but we shouldn't change that for currently existing integration with zencashjs.

`certificate` should be the same between zencashjs and bitcore-lib-zen, but there are small differences between the json representation from zendoo, specifically `sidechainId` instead of `scid` and the unit of the fees.
The [test in bitcore-lib-zen](https://github.com/HorizenOfficial/bitcore-lib-zen/blob/mo/zendoo/test/transaction/sidechain-certificate.js#L31) checks each field individually, I formatted the result then checked for deep equality.